model/src/main/kotlin/utils/SortedSetConverters.kt

```diff
@@ -92,6 +93,10 @@ class SnippetFindingSortedSetConverter : StdConverter<Set<SnippetFinding>, Sorte
     override fun convert(value: Set<SnippetFinding>) = value.toSortedSet(compareBy { it.sourceLocation })
 }
 
+class StringSortedMapConverter : StdConverter<Map<String, String>, SortedMap<String, String>>() {
```

Comment:
Would `@JsonPropertyOrder(alphabetic = true)` work instead?

Reply:
Yes, it does.